bin/status.py

```diff
@@ -34,6 +36,13 @@ def main():
     parser = ArgumentParser()
     parser.add_argument("config", help="json cluster configuration file")
     args = parser.parse_args()
+    if 'config.json' not in args.config:
+        print("Please provide config.json")
+        print("Run : python3 status.py config.json")
+        sys.exit(1)
+    if not path.exists(args.config):
```

Comment:
We should handle this error in the constructor of ClusterConfiguration, then it will work for all files.
However, that method currently throws the right exception: No such file or directory. So the error message looks pretty clear to me.